src/sparql/fix_xref_prefixes.ru

```diff
@@ -2,22 +2,24 @@ prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
 prefix owl: <http://www.w3.org/2002/07/owl#>
 prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
 prefix xref: <http://www.geneontology.org/formats/oboInOwl#hasDbXref>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
 
 DELETE {
   ?entity xref: ?value .
   ?ax a owl:Axiom ;
          owl:annotatedSource ?entity ;
          owl:annotatedProperty xref: ;
          owl:annotatedTarget ?value ;
-         ?p ?v .
+         <http://purl.obolibrary.org/obo/ECO_0000218> ?v .
```

Comment:
Simplifying this query as there is no other metadata we want to copy other than this one.